runtime/kphp-backtrace.cpp

```diff
@@ -14,7 +14,7 @@
 
 std::forward_list<char **> KphpBacktrace::last_used_symbols_;
 
-KphpBacktrace::KphpBacktrace(void **raw_backtrace, int32_t size) noexcept {
+KphpBacktrace::KphpBacktrace(void *const *raw_backtrace, int32_t size) noexcept {
```

Comment:
looks a bit ugly, maybe revert it? up to you though.

Reply:
Without it I can't make `MallocStateHolder::get_last_malloc_replacement_backtrace()` const.
Because it doesn't fit [`backtrace_symbols`](https://linux.die.net/man/3/backtrace_symbols) const signature